documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
> With the amount of work you're describing would be required for docs, that doesn't seem realistic.

It's realistic if we decide today. It's maybe 1, conservatively 2 hours of docs work. It's not realistic, however, if we don't decide until tomorrow. I'm in favor of splitting it, just pick a side.

This is, uh, a weird time to say something like this, but the indecision is a bigger issue than just holding noses and picking one of the two options.